tests/difference/core/driver/core_test.go

```diff
@@ -256,7 +255,6 @@ func (s *CoreSuite) matchState() {
 				s.Require().Errorf(err, diagnostic+" power mismatch for val %d, expect 0 (nil), got %d", j, actual)
 			}
 		}
-		// TODO: outstanding downtime
```

Comment:
Not a good idea/ not worth doing. The method is blackbox and delegations are implementation detail.
More docs/ onboarding material is coming which will explain method and reasoning.